src/Display/Gallery/Gallery.tsx

```diff
@@ -13,12 +13,10 @@ import {
   GalleryThumbnailWrapper,
 } from './GalleryStyle';
 
-const defaultImagesDisplayed = 8;
-
 const Gallery = ({
   initialVisibility,
```

Comment:
The default value should be false.

Reply:
Actually I think it's important that it is `undefined`. Anything else would mean that the component controlled.

Reply:
I still think it should be false. Because for the default state the image doesn't open when the Gallery component is rendered. That means the default value of the prop is false. 
Don't understand what difference showing undefined there makes.

Reply:
Hmm yeah now that I think about it, it doesn't make too much sense not to specify `false` here. Thanks!